### PR TITLE
ER-1350: Added material type

### PR DESCRIPTION
**Pull request by martinyde**
- Add material type selection to form

### CODE REVIEW

File: ereolen_support/ereolen_material.form.inc [hunk 1]
```diff
@@ -41,6 +41,17 @@ function ereolen_material_form($form, &$form_state) {
     '#required' => TRUE,
   ];
 
+  $form['ereolen_material_type'] = [
+    '#type' => 'select',
+    '#title' => t('Type'),
+    '#options' => [
+      t('E-Book') => t('E-Book'),
+      t('Audiobook') => t('Audiobook'),
+      t('E- and audiobook') => t('E- and audiobook'),
```
Comment (rimi-itk) [inline ereolen_support/ereolen_material.form.inc]: I would write this as `E-book and audiobook` (I'm not sure English works the same way as Danish when it comes to using hyphens like this).

File: ereolen_support/ereolen_material.form.inc [hunk 2]
```diff
@@ -41,6 +41,17 @@ function ereolen_material_form($form, &$form_state) {
     '#required' => TRUE,
   ];
 
+  $form['ereolen_material_type'] = [
+    '#type' => 'select',
+    '#title' => t('Type'),
+    '#options' => [
+      t('E-Book') => t('E-Book'),
```
Comment (rimi-itk) [inline ereolen_support/ereolen_material.form.inc]: “E-Book” is usually written as ”E-book” (cf. https://github.com/eReolen/base/blob/develop/sites/all/modules/reol_base/reol_base.module#L719).

**Review by rimi-itk — APPROVED** [review]
Approved with comments on spelling.

File: ereolen_support/ereolen_material.form.inc (hunk 1 above)
Comment (martinyde) [reply ereolen_support/ereolen_material.form.inc]: Changed

File: ereolen_support/ereolen_material.form.inc (hunk 2 above)
Comment (martinyde) [reply ereolen_support/ereolen_material.form.inc]: Changed

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**